.pipelines/iac.yml

```diff
@@ -0,0 +1,45 @@
+trigger: none
+variables:
+  - group: ci-cd-common

```

Comment:
>ci-cd-common [](http://example.com/codeflow?start=11&length=12)

Should this value be parametrized?

Reply:
No. Variable values in the variable groups configure the pipeline behavior. But the variable group name that contains those variables is normally not parametrized.